README.md

```diff
@@ -12,115 +12,131 @@ npx knex migrate:latest
 npx knex seed:run
 cp .env.example .env
 ```
+
 To run in development:
+
 ```sh
 npm run dev
 ```
 
 To run in production:
+
 ```sh
 npm start
 ```
+
 ## User Stories
 
 ### MVP
 
-* As a developer who is new to the project, I want to see an informative README, so that I can get started with the code right away.
-* As a developer, I want the linter and unit tests to run automatically before I make new commits, so that I don't have to remember to run them. (Husky)
-* As a user, I want to join an event, so that I can keep track of the events I am participating in in Coject.
-* As a user, I want to be added to an event when I click the Join button.
+- As a developer who is new to the project, I want to see an informative README, so that I can get started with the code right away.
+- As a developer, I want the linter and unit tests to run automatically before I make new commits, so that I don't have to remember to run them. (Husky)
+- As a user, I want to join an event, so that I can keep track of the events I am participating in in Coject.
+- As a user, I want to be added to an event when I click the Join button.
```

Comment:
❔ Since our project has its own product backlog, we can probably omit this section — but, links to the Product Backlog and the Miro board might be useful here.